applications/nrf_desktop/doc/settings_load.rst

```diff
@@ -0,0 +1,30 @@
+.. _settings_load:
+
+Settings load module
+####################
+
+Use the Settings load module to trigger loading the data from non-volatile memory.
+
+Module Events
+*************
+
++----------------+-------------+-------------------+-----------------+------------------+
+| Source Module  | Input Event | This Module       | Output Event    | Sink Module      |
++================+=============+===================+=================+==================+
+|                |             | ``settings_load`` |                 |                  |
++----------------+-------------+-------------------+-----------------+------------------+
+
+Configuration
+*************
+
+The Settings load module is enabled for every nRF Desktop device with :ref:`zephyr:settings` enabled - when :option:`CONFIG_SETTINGS` is set.
+
+Loading all the settings in one go is more efficitent than making each subsystem load the settings independently.
```

Comment:
This is not true. Also the fact that we load it from one place is because of how Zephyr's settings architecture is done.

I would much prefer to load settings in each module (is parts and only when needed) but it is impossible.

Reply:
Zephyr provides the `settings_load_subtree` function, but using it is ineffective (every time we traverse through the whole settings memory). So there is alternative solution.

Also refer to Zephyr Bluetooth settings documentation: `https://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_BT_SETTINGS.html`

Reply:
The sentence: `Loading all the settings in one go is more efficitent than making each subsystem load the settings independently.` will be removed. It may be misleading. Discussed offline.